corehq/apps/analytics/tasks.py

```diff
@@ -140,7 +141,7 @@ def _get_client_ip(meta):
     return ip
 
 
-def _send_form_to_hubspot(form_id, webuser, cookies, meta, extra_fields=None):
+def _send_form_to_hubspot(form_id, webuser, cookies, meta, extra_fields=None, prelogin=False):
```

Comment:
Of the 8 uses of this function, only 1 is prelogin so it felt weird to have to include extra fields in the vast majority of calls, particularly because it would add some boilerplate to each other call and most subsequent ones. That being said, I dont actually have  strong opinion on this and if you think the other one is cleaner, I am happy to switch it up. (I think this related to the multiple webuser types, changing to a string for consistency requires changing a lot of the past work and passing the rest of the info contained in the user object in separately)